src/test/ui/lint/let_underscore/let_underscore_drop.rs

```diff
@@ -0,0 +1,14 @@
+// check-pass
+// compile-flags: -W let_underscore_drop
```

Comment:
You can accomplish the same behavior with `#![warn(let_underscore_drop)]`

Reply:
Done!